### ISSUE TITLE
survey_point shouldn't be moved

### ISSUE THREAD

**Issue by Bibi56** [openstreetmap/iD#9255]
### URL

https://www.openstreetmap.org/edit?editor=id&node=669983383#map=19/49.75438/4.87604

### How to reproduce the issue?

1. Open the link
2. Read the misleading proposed "correction"

### Screenshot(s) or anything else?

As stated in the note (for survey points in France): _Ne pas déplacer ce point, cf. - **Do not move this node**, see - http://wiki.openstreetmap.org/wiki/WikiProject_France/Repères_Géodésiques#Permanence_des_rep.C3.A8res_
(...)_two geodetic points at the same position (latitude, longitude) should not be merged, their altitudes (ele) being different._

More generally, two points at the same geographical position (latitude, longitude) but different elevation (ele) should not be candidate for a move: they are at the same geographical position by construction/design.

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit

### What version numbers does this issue effect?

2.21.1

### Which browsers are you seeing this problem on?

Firefox

**Comment by 1ec5** [comment]
> More generally, two points at the same geographical position (latitude, longitude) but different elevation (ele) should not be candidate for a move: they are at the same geographical position by construction/design.

A specific exception for survey points makes sense to me, but a more general exception when `ele` differs would be trickier. I also suggested a more general exception back in https://github.com/openstreetmap/iD/issues/6612#issuecomment-508309223. However, sometimes there is a legitimate issue when two coincident points differ in elevation. For example, the GNIS import often put two nodes in the same place, one representing the dam and the other representing the reservoir impounded by it. The difference with survey points is that sometimes they’re _supposed_ to be stacked one atop another, as in your example.

As a workaround, you could set `layer` to `1` and `2`. Perhaps the warning should have a suggestion to tag the feature with a higher or lower `layer`, just as when a road crosses a building. However, `layer` would feel redundant to `ele` for something like this.

Tangentially, I wonder if `man_made=survey_point` should be locked by default so that the user can’t move it by mistake when panning the map, since they’re often mapped with more precision and intentionality than other point features. Holding down Alt (Option) would let the user move it intentionally.

**Comment by Bibi56** [comment]
Yes, locking the position of a `man_made=survey_point`, may be even its removal (if you do it you must know what you're doing) would make sense. No problem for having to add `layer` for the other rare cases.

**Comment by tyrasd** [comment]
Let's close this in favor of a slightly less intrusive solution: #9695
